modules/products/rubymine/src/main/java/com/shopify/shadowenv/products/rubymine/RubyMineRunConfigurationExtension.java

```diff
@@ -68,7 +68,7 @@ private void updateConfig(@NotNull AbstractRubyRunConfiguration<?> configuration
         env.clear();
         env.putAll(m);
 
-        var curName = configuration.getSdk().getName();
+        String curName = configuration.getSdk().getName();
```

Comment:
The var 💥 for me locally; but maybe I have an old JDK set up. Any advice on minimum supported JDK?